app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -29,7 +29,7 @@ def audio?(thumb)
 
       def fetch_thumbnail(object)
         return object if object.thumbnail_id == object.id
-        Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: object.thumbnail_id)
+        Hyrax.query_service.find_by(id: object.thumbnail_id)
```

Comment:
👍🏻

Reply:
is it worth submitting this part now, while we work on the rest?

Reply:
i ended up backing that change out https://github.com/samvera/hyrax/pull/5626/commits/8ed796fa735540dc9d43541d78a8370c72d71868 because this new uploads path skips that code entirely